lib/groonga-query-log/server-verifier.rb

```diff
@@ -122,8 +124,13 @@ def run_reporter
         @options.open_output do |output|
           loop do
             result = @different_results.pop
-            break if result.nil?
-            report_result(output, result)
+            unless result.nil?
+              report_result(output, result)
+            end
+            result = @error_results.pop
+            unless result.nil?
+              report_result(output, result)
+            end
```

Comment:
Normally, we should not wait multiple queues. Error queue is never popped different queue is finished.